packages/app/src/cli/models/extensions/specifications/app_config_webhook_subscription.ts

```diff
@@ -50,14 +50,15 @@ function transformFromWebhookSubscriptionConfig(content: object) {
   const {api_version, subscriptions = []} = webhooks
 
   const webhookSubscriptions = subscriptions.flatMap((subscription) => {
-    const {uri, topics, ...optionalFields} = subscription
+    // compliance_topics gets handled by privacy_compliance_webhooks
+    const {uri, topics, compliance_topics: _, ...optionalFields} = subscription
     if (topics)
       return topics.map((topic) => {
         return {api_version, uri, topic, ...optionalFields}
       })
   })
 
-  return webhookSubscriptions.length > 0 ? webhookSubscriptions : []
+  return webhookSubscriptions.length > 0 ? webhookSubscriptions.filter(Boolean) : []
```

Comment:
fancy way of filtering out `undefined` values we get from `subscription` objects that might not have `topics` (ie. in the case of privacy compliance webhooks)